tests/unit/EntityInfo.spec.ts

```diff
@@ -77,28 +77,129 @@ const mockEntityInfo = [
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Benefit Company',
+    numberedDesc: '',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
+  },
+  {
+    entityType: CorpTypeCd.BEN_CONTINUE_IN,
+    tombstone: {
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Benefit Company',
+    numberedDesc: 'Numbered Benefit Company',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
+  },
+  {
+    entityType: CorpTypeCd.CCC_CONTINUE_IN,
+    tombstone: {
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Community Contribution Company',
+    numberedDesc: '',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
+  },
+  {
+    entityType: CorpTypeCd.CCC_CONTINUE_IN,
+    tombstone: {
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Community Contribution Company',
+    numberedDesc: 'Numbered Community Contribution Company',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
+  },
+  {
+    entityType: CorpTypeCd.ULC_CONTINUE_IN,
+    tombstone: {
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Unlimited Liability Company',
+    numberedDesc: '',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
+  },
+  {
+    entityType: CorpTypeCd.ULC_CONTINUE_IN,
+    tombstone: {
+      filingType: FilingTypes.CONTINUATION_IN
+    },
+    description: 'BC Unlimited Liability Company',
+    numberedDesc: 'Numbered Unlimited Liability Company',
+    tempId: 'T1234567',
+    nameRequest: {
+      nrNum: 'NR 1234567'
+    },
+    nameRequestApprovedName: 'Xyz Ltd.'
   }
 ]
 
-//
-// FUTURE: These tests generate a lot of
-// "UnhandledPromiseRejectionWarning:NavigationDuplicated"
-// and
-// "UnhandledPromiseRejectionWarning: Unhandled promise rejection"
-// warnings, which should be cleaned up.
-//
 for (const mock of mockEntityInfo) {
   describe(`Entity Info component for a ${mock.entityType} with a NR`, () => {
     let wrapper: any
 
     beforeEach(() => {
-      wrapper = wrapperFactory(EntityInfo, null, {
-        tombstone: mock.tombstone,
-        entityType: mock.entityType,
-        tempId: mock.tempId,
-        nameRequest: mock.nameRequest,
-        nameRequestApprovedName: mock.nameRequestApprovedName
-      }, 'incorporation-define-company')
+      wrapper = shallowWrapperFactory(
+        EntityInfo,
+        null,
+        {
+          tombstone: mock.tombstone,
+          entityType: mock.entityType,
+          tempId: mock.tempId,
+          nameRequest: mock.nameRequest,
+          nameRequestApprovedName: mock.nameRequestApprovedName
+        }
+      )
```

Comment:
I changed this (and below) to shallow-mount because we don't need the child components to render in these tests.